javascript/chronometer.js

```diff
@@ -1,10 +1,68 @@
 class Chronometer {
   constructor() {
+    
     // ... your code goes here
+  
+   
+        this.currentTime = 0;
+        this.intervalId = null ;
+      }
+    
+      start(callback) {
+        this.intervalId=setInterval(()=> {
+         callback()
```

Comment:
here you need the condition:
```
if (callback) { 
callback()
}
```
after writing this if condition for the callback function, all tests are passing